molgenis/molgenis-emx2
# build: add apps/build.gradle caching

how to test:
- on master do a 'gradle run' and then a second time. 
- then checkout this branch and do it again
- before this PR you would see the javascript build execute twice, in this branch it is skipped in second time.

todo:
- [na] added tests
- [x] remove the check format git hook

to test: 
- [x] does the js cache build work locally 
- [x] is the cache busted on change locally
- [x] are js test run locally
- [x] are js tests run on ci 
- [x] are e2e tests run on ci 
- [x] are the node severs being build on ci
- [x] are the playgrounds being build locally 
- [x] are the playgrounds being build on ci ( components-lib menu is not working)
